interpretation/deepseismic_interpretation/dutchf3/data.py

```diff
@@ -584,22 +578,20 @@ class TrainPatchLoader(PatchLoader):
 
     def __init__(
         self,
-        data_dir,
-        n_classes,
+        config,
         split="train",
-        stride=30,
-        patch_size=99,
+        # stride=30,
+        # patch_size=99,
         is_transform=True,
         augmentations=None,
         seismic_path=None,
         label_path=None,
         debug=False,
     ):
         super(TrainPatchLoader, self).__init__(
-            data_dir,
-            n_classes,
-            stride=stride,
-            patch_size=patch_size,
+            config,
+            # stride=stride,
+            # patch_size=patch_size,
```

Comment:
Same comment as above: 
I suggest we delete code we don't need instead of having it commented out.